release-3.7.0-announcement.md

```diff
@@ -0,0 +1,31 @@
+We're excited to announce the availability of Jamulus 3.7.0!
+
+This release re-organizes Server lists to make room for more servers:
+- Default has been renamed to Any Genre 1.
+- Any Genre has been renamed to Any Genre 2.
+- Any Genre 3 has been added.
+- Choral/Barbershop has been added.
+- Classical/Folk/Choral has been renamed to Classical/Folk.
+
+Besides that, tens of usability improvements, enhancements, bugfixes and optimizations have been integrated.
+Please find all the details in the [Changelog](https://github.com/jamulussoftware/jamulus/releases/tag/r3_7_0).
+
+Downloads:
```

Comment:
Maybe you could add that the binaries are now hosted on GH and SF?

Reply:
I think we do need to make a decision about SF though because having two download links means the UX has to "fork" in some way, I think, eg "download here" > landing page with links, not simply "download windows here".

Reply:
I think the information is both relevant and is at risk of being confusing. When I think of the average choir member or non-technical jamming person I think they will only care about how to download. The place where we host binaries is an implementation detail. They won't know Github and they won't know SourceForge other than "was this the confusing site in English where I didn't know where to click". But I think it's still worth adding a note as several people used SourceForge as their primary download address and might check for new releases there.

I have added a short note in braces. Would this work for you?

Reply:
BTW if we did - and only if we did - want to "sunset" SF, we should keep an eye on the download counts from time to time after this.